Mathlib/Topology/Category/TopCat/Limits/Konig.lean

```diff
@@ -89,86 +62,89 @@ private abbrev FiniteDiagram (J : Type u) [SmallCategory J] :=
-      ⟨f.1, f.2.1, Finset.mem_union_right _ f.2.2.1, Finset.mem_union_right _ f.2.2.2.1, f.2.2.2.2⟩
-    refine' ⟨⟨A.1 ⊔ B.1, A.2.image ιA ⊔ B.2.image ιB⟩, _, _⟩
-    · rintro u hu f hf
-      have : ιA f ∈ A.2.image ιA ⊔ B.2.image ιB := by
-        apply Finset.mem_union_left
-        rw [Finset.mem_image]
-        refine' ⟨f, hf, rfl⟩
-      exact hu this
-    · rintro u hu f hf
-      have : ιB f ∈ A.2.image ιA ⊔ B.2.image ιB := by
-        apply Finset.mem_union_right
-        rw [Finset.mem_image]
-        refine' ⟨f, hf, rfl⟩
-      exact hu this
+  intro A B
+  let ιA : FiniteDiagramArrow A.1 → FiniteDiagramArrow (A.1 ⊔ B.1) := fun f =>
+    ⟨f.1, f.2.1, Finset.mem_union_left _ f.2.2.1, Finset.mem_union_left _ f.2.2.2.1, f.2.2.2.2⟩
+  let ιB : FiniteDiagramArrow B.1 → FiniteDiagramArrow (A.1 ⊔ B.1) := fun f =>
+    ⟨f.1, f.2.1, Finset.mem_union_right _ f.2.2.1, Finset.mem_union_right _ f.2.2.2.1, f.2.2.2.2⟩
+  refine' ⟨⟨A.1 ⊔ B.1, A.2.image ιA ⊔ B.2.image ιB⟩, _, _⟩
+  · rintro u hu f hf
+    have : ιA f ∈ A.2.image ιA ⊔ B.2.image ιB := by
+      apply Finset.mem_union_left
+      rw [Finset.mem_image]
+      refine' ⟨f, hf, rfl⟩
+    exact hu this
+  · rintro u hu f hf
+    have : ιB f ∈ A.2.image ιA ⊔ B.2.image ιB := by
+      apply Finset.mem_union_right
+      rw [Finset.mem_image]
+      refine' ⟨f, hf, rfl⟩
+    exact hu this
 #align Top.partial_sections.directed TopCat.partialSections.directed
 
-theorem partialSections.closed [I : ∀ j : J, T2Space (F.obj j)] {G : Finset J}
+theorem partialSections.closed [∀ j : J, T2Space (F.obj j)] {G : Finset J}
     (H : Finset (FiniteDiagramArrow G)) : IsClosed (partialSections F H) := by
   have :
     partialSections F H =
-      ⋂ (f : FiniteDiagramArrow G) (_ : f ∈ H), { u | F.map f.2.2.2.2 (u f.1) = u f.2.1 } := by
+      ⋂ (f : FiniteDiagramArrow G) (_ : f ∈ H), {u | F.map f.2.2.2.2 (u f.1) = u f.2.1} := by
     ext1
     simp only [Set.mem_iInter, Set.mem_setOf_eq]
     rfl
   rw [this]
   apply isClosed_biInter
   intro f _
   -- Porting note: can't see through forget
-  have : T2Space ((forget TopCat).obj (F.obj f.snd.fst)) := I f.snd.fst
-  -- Porting note: used to be continuity
+  have : T2Space ((forget TopCat).obj (F.obj f.snd.fst)) :=
+    inferInstanceAs (T2Space (F.obj f.snd.fst))
   apply isClosed_eq
-  · apply Continuous.comp (F.map f.snd.snd.snd.snd).continuous_toFun (continuous_apply f.fst)
-  · apply continuous_apply f.snd.fst
+  -- Porting note: used to be a single `continuity` that closed both goals
+  · exact (F.map f.snd.snd.snd.snd).continuous.comp (continuous_apply f.fst)
+  · continuity
 #align Top.partial_sections.closed TopCat.partialSections.closed
 
 /-- Cofiltered limits of nonempty compact Hausdorff spaces are nonempty topological spaces.
 -/
-theorem nonempty_limitCone_of_compact_t2_cofiltered_system [IsCofilteredOrEmpty J]
+-- porting note: generalized from `TopCat.{u}` to `TopCatMax.{u,v}`
```

Comment:
In master this is `TopCat.{max u v}`, but presumably using `TopCatMax` is more correct here. In lean 3 it was just `TopCat.{u}`.